pom.xml

```diff
@@ -658,7 +658,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.module</groupId>
         <artifactId>jackson-module-scala_${scala.binary.version}</artifactId>
-        <version>${fasterxml.jackson.databind.version}</version>
+        <version>${fasterxml.jackson.version}</version>
```

Comment:
I made this change that referenced
https://github.com/apache/spark/blob/0ed9fae45769d4b06b8cf8128f462f09ff3d9a72/pom.xml#L712